app/views/password_mailer/change_password.html.erb

```diff
@@ -29,7 +29,7 @@
 </td>
                       <td bgcolor="#e9573f">
                         <div class="text-btn" style="color:#ffffff; font-family:Arial, sans-serif; min-width:auto !important; font-size:16px; line-height:20px; text-align:center">
-                          <a href=<%= edit_user_password_url(@user, token: @user.confirmation_token.html_safe) %> target="_blank" rel="noopener" class="link-white" style="color:#ffffff; text-decoration:none"><span class="link-white" style="color:#ffffff; text-decoration:none">CHANGE PASSWORD</span></a>
+                          <a href="<%= edit_password_url(token: @user.confirmation_token.html_safe) %>" target="_blank" rel="noopener" class="link-white" style="color:#ffffff; text-decoration:none"><span class="link-white" style="color:#ffffff; text-decoration:none">CHANGE PASSWORD</span></a>
```

Comment:
use a `link_to` helper instead of interpolating the href

Reply:
Any idea why it was like this originally? I was hesitant to change something strange without knowing why it was strange in the first place.

Reply:
I'm incline to fix these all in a separate PR. There's a bunch of ugly in the email templates and I don't really want to get into it.